js/components/CategoriesPage.jsx

```diff
@@ -4,104 +4,47 @@ var $ = require('jquery');
 var AuthenticatedRoute = require('../mixins/AuthenticatedRoute.jsx');
 var CategoriesList = require('./CategoriesList.jsx');
 var CategoriesPageFilter = require('./CategoriesPageFilter.jsx');
+var CategoriesActionCreator = require('../actions/CategoriesActionCreator.js');
+var CategoriesStore = require('../stores/CategoriesStore.js');
 var Footer = require('./Footer.jsx');
 var React = require('react');
 var strings = require('../lib/strings_utils.js');
 var Toolbar = require('./Toolbar.jsx');
 
+function getStateFromStores() {
```

Comment:
This pattern of creating the function outside the React class comes from the Facebook people as does the '_' convention.

I have faith that there's a reason they use it.